### ISSUE TITLE
Revise a page on the CC public wiki

### ISSUE THREAD

**Issue by kgodey** [creativecommons/faq#35]
_This is an Outreachy contribution task. Multiple people may complete this task, it will not be closed until the Outreachy winter 2019 contribution period ends._

---
The [Creative Commons Wiki](https://wiki.creativecommons.org/) has a lot of old/inaccurate content. Revise one wiki page and post a link to your contribution here. Please ask questions about any content you are unsure of.

To get edit access to the wiki, please sign up for a [CCID here](https://login.creativecommons.org/register.php) and tell us about it in `#cc-gsoc-outreachy` on Slack. We will give your account edit permissions.

**Comment by alexiaOkarabi** [comment]
Hello @kgodey I have signup using the link you provided above. Would love to work on this issue.

**Comment by kgodey** [comment]
@alexiaOkarabi please tell me what your CCID is so that I can give you edit permissions.

**Comment by mchesler613** [comment]
I also have a CCID msc2019, @kgodey

**Comment by mmoreshead** [comment]
Hi @mchesler613 - I've approved your wiki account so you can start editing!

**Comment by tyanastone** [comment]
Hello, I created an account. My CCID is **tyana**

**Comment by Angelvicks** [comment]
Hello @kgodey , am an outreachy applicant and  i will to contribute to work on this issue. My CCID is angetaffo

**Comment by RegMWilson** [comment]
Hi I am an Outreachy applicant. I would love to be involved in this issue and I have created a CCID: **RegIsFred**     Looking forward to being involved.

**Comment by Angelvicks** [comment]
I succeeded in revising one wiki page, i mostly revise the French section since it was still in English while the title there was French. The link is below, https://wiki.creativecommons.org/wiki/Translate/Global_Network_Strategy

**Comment by Rubaaw** [comment]
Hello @kgodey 
Would love to work on this issue, my CCID is **Ruba**

**Comment by IvetaHron** [comment]
Hello @kgodey, I am Outreachy applicant and I would like to work on this issue. My CCID is iveta.

**Comment by mmoreshead** [comment]
@IvetaHron and @Rubaaw - you will need to log into the wiki once before I can give you permissions to edit the wiki. Can you let me know once you've done that?

@Angelvicks @RegMWilson @tyanastone -- you're all good to go.

**Comment by Rubaaw** [comment]
@mmoreshead yes im already signed in :D

**Comment by mmoreshead** [comment]
@Rubaaw when you are logged in, can you confirm for me what is up on the right top corner of the page?

**Comment by mbengtanyi** [comment]
Revised the [documentation wiki page](https://wiki.creativecommons.org/wiki/Documentation).
- Added link to wiki signup.
- Took out redundant data and dead links about CC Learns.
- Included information about CC Education/OER.
- Minor fixes: punctuation.
[Documentation history](https://wiki.creativecommons.org/index.php?title=Documentation&action=history)

**Comment by Rubaaw** [comment]
CCID-Ruba  Talk  Preferences  Watchlist  contributions. LOG OUT.

**Comment by moreinob** [comment]
Hi @kgodey, I would love to contribute on this. My CCID is moreinob. Thanks

**Comment by Angelvicks** [comment]
Hello, I did some punctuation and grammar update on this page https://wiki.creativecommons.org/wiki/Developers

**Comment by mmoreshead** [comment]
@Rubaaw & @IvetaHron  - great, you have edit permission now.

@moreinob can I ask you to log into the wiki one more time? I can't see your CCID yet and sometimes it takes an additional log in.

**Comment by moreinob** [comment]
@mmoreshead my CCID is moreinouma not moreinob, I mistakenly used my ID here. Please use moreinouma

**Comment by mmoreshead** [comment]
ok @moreinob - I have updated your permissions!

**Comment by Rubaaw** [comment]
Hello,
So i was revising and reading the wiki, there is [OER wiki ](https://wiki.creativecommons.org/wiki/OER) that has only a link redirecting to [https://creativecommons.org/education](https://creativecommons.org/education)
Are we allowed to add new content to the wiki ? or just revising the existed content ?

**Comment by Rubaaw** [comment]
I have revised [Translate Wiki page](https://wiki.creativecommons.org/wiki/Translate).

- Reorder the table Format (Lang. Column width)
- Added hyperlinks to the Languages.
- Added missing languages with links
- Removed dead links

**Comment by Angelvicks** [comment]
I have this page https://wiki.creativecommons.org/wiki/Translate/Documentation#CC_Learn_Productions
I removed dead links.

**Comment by kgodey** [comment]
thank you for your contributions, everyone!

**Comment by gabrielassena** [comment]
Hello, my name is Gabriela and I'm a Outreachy applicant and I will be glad to make a contribution to this project, I created my CCID(gabrielassena), can you please get me edit permission

Thanks

**Comment by mmoreshead** [comment]
Hi @gabrielassena - you will need to log into the wiki one more time before your CCID can be approved. Please log in and confirm here afterwards.

**Comment by gabrielassena** [comment]
Hi @mmoreshead - I logged in again, I think the CCID is available for approval.
Tks.

**Comment by mmoreshead** [comment]
Perfect, you have permissions now @gabrielassena

**Comment by GettyOrawo** [comment]
Hey, I just signed up and I don't understand what exactly is my CCID. Some help please?

**Comment by annatuma** [comment]
@GettyOrawo have you been able to log into the Wiki yet? I assume you signed up [here](https://login.creativecommons.org/register.php) for a CCID. You'll need to log in at least once before @mmoreshead can approve your CCID for edit permissions.

**Comment by mmoreshead** [comment]
@GettyOrawo when you log into the wiki, you can see your CCID on the top right side.

**Comment by Jethet** [comment]
@kgodey My CCID is Jethet. I would like to contribute to the wiki. Re. the process: do I just pick a page myself or do you assign pages?

**Comment by mmoreshead** [comment]
@Jethet I just approved your account.

**Comment by Jethet** [comment]
Thank you @mmoreshead . I am new to your way of working so it would be great to get a bit more explanation about what to do next. Thanks!

**Comment by Jethet** [comment]
@mmoreshead  More specifically: is this the information we need to adhere to? https://github.com/creativecommons/cc-resource-archive and https://github.com/creativecommons/cc-resource-archive/blob/master/resourcetemplate.md

**Comment by Jethet** [comment]
@mmoreshead  Could you have a look to see if this is in line with what is expected:
https://wiki.creativecommons.org/wiki/Wiki

**Comment by mmoreshead** [comment]
@Jethet simply edit a page, no need to do anything complicated.

**Comment by GettyOrawo** [comment]
![Screenshot from 2019-10-22 07-37-38](https://user-images.githubusercontent.com/17080976/67259547-145b0e80-f49f-11e9-8579-1de47e6f5d70.png)

Here's the page I'm bumping into once I log in. Sorry, so what exactly is my CCID?

**Comment by mmoreshead** [comment]
@GettyOrawo that's the login page, the info I'm looking for is on the wiki itself (when you are logged in) https://wiki.creativecommons.org/wiki/Main_Page  -- What does it say after CCID on the top right side there when you are logged in?

**Comment by Jethet** [comment]
@mmoreshead  I am revising this page https://wiki.creativecommons.org/wiki/Data. Under the third question of the section _Frequently asked questions about data and CC licenses_ it says:
> a database provider should first make sure she has all rights necessary to do so.
Are there guidelines for the use of her/his/their and she/he/them in the CC wiki?

**Comment by mmoreshead** [comment]
@Jethet No, at this time there are no guidelines.

**Comment by Jethet** [comment]
@mmoreshead  I have revised https://wiki.creativecommons.org/wiki/Data.
The main edits aim to:

1. Clarify the text, avoid using words such as 'notwithstanding' and 'however
2. Increase readability by breaking up long sentences into two proper sentences
3. Solve unclear sentences such as 
> Database providers should also consider carefully what elements of the database she wants covered by the CC legal tool 

Some very specific text such as "bright line test" have been replaced by text that will be better understood by non-Americans. Also, some grammar and writing errors were corrected.

**Comment by Jethet** [comment]
@mmoreshead 
I am editing the page regarding the OER project (https://wiki.creativecommons.org/wiki/OER_Project) and have added a general explanation of what OER means, linking it to another page of the CC wiki with more information.

In the current text on the page for the OER Project, it says:

> There are many ways you can help develop the OER Project and the open educational community

The 'project' part of the OER Project is not explained anywhere. Where can I find information about the project, the goal, who is involved, some history, etc., to clarify this?

**Comment by mmoreshead** [comment]
@Jethet This would be a very good example of a page that is so long out of date that it's probably not worth updating. Maybe try another page?

**Comment by Jethet** [comment]
Thank you. I will leave it at that.

**Comment by GettyOrawo** [comment]
my CCID is Getty Orawo could I kindly get permissions to edit? Thank you

**Comment by mmoreshead** [comment]
@GettyOrawo there are no spaces in a CCID. As I already said, log into the wiki and tell me what it says after CCID on the top right side.

**Comment by GettyOrawo** [comment]
Hey @mmoreshead sorry but it honestly says Getty Orawo unless I am to add an underscore?

![Screenshot from 2019-10-30 22-53-49](https://user-images.githubusercontent.com/17080976/67893865-7c2ceb80-fb68-11e9-9029-8489db5a45e6.png)
![Screenshot from 2019-10-30 22-54-52](https://user-images.githubusercontent.com/17080976/67893867-7cc58200-fb68-11e9-8a9f-ef2091a92d91.png)

**Comment by GettyOrawo** [comment]
This is the URL when I click on the CCID, could be the correct CCID maybe? **https://wiki.creativecommons.org/wiki/User:CCID-Getty_Orawo**

**Comment by mmoreshead** [comment]
@GettyOrawo your account is approved now

**Comment by GettyOrawo** [comment]
Thank you much.

**Comment by monique1087** [comment]
Hello, @mmoreshead and @kgodey ! I would love to contribute with this project. I already make an account at the CCID and just have to approve 👍  I want to translate to portuguese/br the "CC network". Can I do that, please? ^^

**Comment by mmoreshead** [comment]
@monique1087 What is your CCID? Log into the wiki and look up in the right hand corner.

**Comment by GettyOrawo** [comment]
Hey Team. I revamped the Case studies Wiki. Taking into account that the information provided was not enough to make someone intrigued to contribute to the betterment of the case studies, I added more detailed descriptions on how to contribute and a brief preview of what the book by CC Australia is about. 
https://wiki.creativecommons.org/wiki/Case_Studies

![Screenshot from 2019-11-05 06-48-23](https://user-images.githubusercontent.com/17080976/68177113-5785a900-ff98-11e9-9a9c-97396e6bb4b8.png)
